filters/data-test.properties

```diff
@@ -48,8 +48,10 @@ environment.build.hibernate.connection.validationQuery=select 1 from INFORMATION
 
 # uPortal server configuration properties
 environment.build.uportal.server=localhost:8080
+environment.build.real.uportal.server=localhost:8080
 environment.build.uportal.protocol=http
 environment.build.uportal.context=/uPortal
+environment.build.real.uportal.context=/uPortal
 environment.build.uportal.secureSessionCookie=false
```

Comment:
Can we reach the goals of this change without changing the filters files?

I'm concerned that the difference between these properties is not readily apparent by looking at them.  The filters files are something every adopter has to look at and understand... is it possible to support this feature without introducing additional, complexity to the configuration?